modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -545,6 +550,115 @@ public void testAddObjectActionWithConditionExpression() throws Exception {
 		_objectActionLocalService.deleteObjectAction(objectAction);
 	}
 
+	@Test
+	public void testAddObjectActionWithCycle() throws Exception {
```

Comment:
I think we can add this scenario in the method `testAddObjectAction()`.
We can add a comment before this scenario.
What do you think?